### PR TITLE
payout withdrawal event

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [ ] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [ ] Automatic tests pass
- [ ] Sanity check passed
- [ ] Self-review passed
- [ ] Version updated

### CODE REVIEW

**Comment by Anze1m** [comment]
### Don't merge it yet please

File: contracts/Storage/RewardsEventEmitter.sol [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import "../Boilerplate/Head.sol";
+
+
+contract RewardsEventEmitter is Base {
+
+    constructor(Head _head) Base(_head) public {}
+
+    event AtlasPayoutWithdrawal(address target, address beneficiary, uint256 value);
+    event ApolloBundleFeePayout(address uploader, bytes32 bundleId, uint256 value);
```
Comment (krzysztof-jelski) [inline contracts/Storage/RewardsEventEmitter.sol]:
```suggestion
    event ApolloBundleReward(address uploader, bytes32 bundleId, uint256 value);
```

Comment (krzysztof-jelski) [reply contracts/Storage/RewardsEventEmitter.sol]: What about this suggestion?
